### ISSUE TITLE
Fishing will sometimes fail regardless of how well you do during the minigame.

### ISSUE THREAD

**Issue by Hardly3D** [tgstation/tgstation#70067]
## Reproduction:

tl;dr
Buy fishing toolbox and equip the rod with hook, reel, and bait
Go fishing (chasm, portal, beach gateway)
Succeed the fishing minigame by filling the bar several times only for it to somehow fail, with the fail balloon message showing up, "It got away".

During the linked round ID, I went chasm fishing to rescue a miner along with a scientist willing to help, we'd mostly get nothing but "It got away" when filling the bar during the minigame, I was using a bone fishing rod equipped with tribal reels and hooks while the scientist was using a regular one, with the bait being deluxe. even then, it took us 30~ minutes in order to get the miner's body back due to us mostly getting "It got away", regardless of how well we did.

I'm not sure if this is a feature™ or actually a bug, but this makes fishing a lot frustrating and time consuming, especially if you're fishing for a dead miner.

**Comment by san7890** [comment]
@AnturK any ideas here?

**Comment by AnturK** [comment]
Not sure, does this happen outside of chasms ? Might be they mess with los checks.

**Comment by tGyro** [comment]
Still happening as of 22/10/2022. 

Round ID: 192851

Normal fishing rod in the chasm to rescue miner. "It Got away" popped up no matter how the minigame went.

**Comment by necromanceranne** [comment]
It's not just sometimes, it's impossible to succeed with the standard rod. Mostly likely something is causing fishing to always fail.

**Comment by Hardly3D** [comment]
New round ID, 196596
Around four people spent over 30 minutes trying to fish someone out, all using rescue hooks and master fishing rods IIRC, admins eventually gave up and eventually just teleported the body out.

**Comment by Ghommie** [comment]
I believe this has been fixed following the PR that removed the ui being closed too early upon completion.
